.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "vendor/agda-stdlib"]
+	path = vendor/agda-stdlib
+	url = git@github.com:conal/agda-stdlib
+[submodule "vendor/agda-categories"]
```

Comment:
What's "vendor" about?

Reply:
This is just a typical Git naming convention when a project must include 3rd party source code in order to be able to build. For example, there is a utility out in the wild called `git-vendor` for helping to manage such things: https://github.com/brettlangdon/git-vendor.

But this subdirectory could be named anything you like.